<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>importPhonebook</class>
  <widget class="QWidget" name="importPhonebook" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>743</width>
        <height>475</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item rowspan="1" row="0" column="0" colspan="2" >
        <widget class="Q3ButtonGroup" name="importFrom" >
          <property name="title" >
            <string>Import Options</string>
          </property>
          <layout class="QGridLayout" >
            <item row="2" column="0" >
              <layout class="QHBoxLayout" >
                <item>
                  <widget class="KUrlRequester" name="fileUrl" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="midLineWidth" >
                      <number>0</number>
                    </property>
                    <property name="url" stdset="0" >
                      <string/>
                    </property>
                    <property name="filter" >
                      <string>text/x-vcard</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="KPushButton" name="abcLoad" >
                    <property name="enabled" >
                      <bool>false</bool>
                    </property>
                    <property name="text" >
                      <string>L&amp;oad</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="1" column="0" >
              <widget class="QRadioButton" name="radioButton1" >
                <property name="text" >
                  <string>Import contacts from &amp;a file:</string>
                </property>
                <property name="buttonGroupId" >
                  <number>1</number>
                </property>
              </widget>
            </item>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="radioButton2" >
                <property name="text" >
                  <string>Import contacts from &amp;KDE AddressBook</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="buttonGroupId" >
                  <number>0</number>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="QLabel" name="formatLabel" >
                <property name="text" >
                  <string/>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="3" column="0" colspan="2" >
        <widget class="QCheckBox" name="clearPhoneBook" >
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <property name="text" >
            <string>&amp;Clear the phonebook before importing (dangerous)</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="K3ListView" name="lv_abc" >
          <property name="selectionMode" stdset="0" >
            <enum>Extended</enum>
          </property>
          <property name="allColumnsShowFocus" >
            <bool>true</bool>
          </property>
          <property name="resizeMode" >
            <enum>AllColumns</enum>
          </property>
          <column>
            <property name="text" >
              <string>Name</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Number</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
          <column>
            <property name="text" >
              <string>Memory</string>
            </property>
            <property name="clickable" >
              <bool>true</bool>
            </property>
            <property name="resizable" >
              <bool>true</bool>
            </property>
          </column>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Contacts to Import:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="1" >
        <layout class="QVBoxLayout" >
          <item>
            <spacer name="spacer5" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>160</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="KPushButton" name="toSim" >
              <property name="text" >
                <string>&amp;SIM</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="toPhone" >
              <property name="text" >
                <string>Pho&amp;ne</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="toDataCard" >
              <property name="text" >
                <string>&amp;DataCard</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="KPushButton" name="dontimport" >
              <property name="text" >
                <string>Don't import</string>
              </property>
              <property name="shortcut" >
                <string/>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >k3listview.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
</ui>
